GiniVision/Classes/GiniVisionDocument.swift

```diff
@@ -71,6 +71,11 @@ extension GiniVisionDocument {
         return 10 * 1024 * 1024
     }
     
+
+    fileprivate var customDocumentValidations: ((GiniVisionDocument) throws -> ())? {
+        return GiniConfiguration.sharedConfiguration.customDocumentValidations
+    }
+    
     // MARK: File validation
     /**
      Validates a document, checking if it has the correct size and type.
```

Comment:
For the `- Throws: ` section (cannot select the line here) it would be enough to say, that `DocumentValidationError` is thrown if the document is not valid. Going into details leads to problems like here - implementation was extended without updating the documentation.